go.mod

```diff
@@ -0,0 +1,83 @@
+module github.com/mainflux/edge
+
+go 1.21.0
+
+replace github.com/mainflux/mainflux => github.com/mainflux/mainflux v0.0.0-20230811131717-9dbe87f03a1a
```

Comment:
replace

Reply:
what do you mean

Reply:
rather remove it. Why do we need a specific version of mainflux?